bartender/local_plugins/plugin_runner.py

```diff
@@ -41,6 +42,10 @@ def __init__(self, entry_point, system, instance_name, path_to_plugin, web_host,
         self.url_prefix = url_prefix
         self.ca_verify = ca_verify
         self.ca_cert = ca_cert
+        if 'log_level' in kwargs:
+            self.plugin_default_log_level = kwargs['log_level']
+        else:
+            self.plugin_default_log_level = logging.INFO
```

Comment:
Should we do `self.plugin_default_log_level = kwargs.get('log_level', logging.INFO)`

Reply:
Yes. I did this while I was troubleshooting something.